ivoatexmeta.tex

```diff
@@ -0,0 +1,6 @@
+% GENERATED FILE -- edit this in the Makefile
```

Comment:
I think this file should be ignored

Reply:
Hum. Without changing ivoatexmeta.tex I don't think the date and draft title can change ? Can it ?

Reply:
If this file is not present it is generated by the make command using the metadata in `Makefile` so I'd recommend making these changes there and adding dropping this file.